### ISSUE TITLE
Embedded browser in Thunderbird does not read at all with NVDA

### ISSUE THREAD

**Issue by Neurrone** [nvaccess/nvda#15482]
### Steps to reproduce:

1. Add a new GMail account (or any email provider) to Thunderbird
2. A browser window asking you to log into Gmail appears. Once logged in, you're asked to grant Thunderbird access.
### Actual behavior:

The browser window that gets opened is completely inaccessible to NVDA, which just reads "unknown". Tabbing or arrowing around produces no feedback.

### Expected behavior:

NVDA reads this browser window normally.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Version: alpha-29111,b341bf97 (2024.1.0.29111)

#### Windows version:
Windows 11 22h2

#### Name and version of other software in use when reproducing the issue:
Thunderbird 102.15.1

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

**Comment by XLTechie** [comment]
@Neurrone You ignored the question on the template:

Does the issue still occur after you run the COM Registration Fixing Tool in 
NVDA's tools menu?

Running that tool has a very good chance of solving this problem. Why have you 
not done so?

**Comment by Neurrone** [comment]
Thanks, that indeed fixed it for me.

I think I just habitually ignore this step because it has never worked for me before.
